docs/modules/operation/pages/performance-data-collection/collectors/prometheus.adoc

```diff
@@ -127,3 +127,16 @@ Here's another excerpt where we extract metrics grouped by CPU:
 ----
 
 This group definition matches metrics called 'node_cpu', groups them by the value of the `cpu` label and extracts the name of the `mode` for the name of the numeric attributes.
+
+
+== Prometheus Node Exporter Collectd Service Configuration Example
```

Comment:
```suggestion
== Prometheus Node Exporter Collectd service configuration example
```
We are starting to move towards using sentence capitalization for headings. In this case, though, we still have to capitalize Prometheus Node Exporter and Collectd, as we are using them as proper nouns in the docs.